### ISSUE TITLE
SPM commit changes "under the hood"

### ISSUE THREAD

**Issue by mathaeus** [AppsFlyerSDK/AppsFlyerFramework#202]
Hi, I'm not sure why this is happening, but I guess you moved the git tag to another commit, when you created the release today.

I'm not sure if you re-released `6.5.4` or if we on our side updated the SDK to your "non-release tag of `6.5.4`" before you officially released `6.5.4`. We are auto-updating minor/patch versions on our side, and if there was no official release from your side yet then I guess this one slipped through on our side.

Long story short: I would be interested in if you regularly tag commits before creating an official release (which refers to a potentially different commit then)? So that we can prevent such "under the hood changes in the end.

Thanks!

<img width="628" alt="Screenshot 2022-04-08 at 15 31 58" src="https://user-images.githubusercontent.com/5389635/162446271-2873bd6b-be61-4f7a-8f05-162814b4a87f.png">

**Comment by af-fess** [comment]
Hi @mathaeus , yes, we added 2 zip files of xcframework to releases manually. We investigate this case what Github API didn't work smoothly.

Thank you for checking this out.

**Comment by timmoosezen** [comment]
This one has been plaguing our team as well.

**Comment by amit-kremer93** [comment]
Fixed
